src/settings.ts

```diff
@@ -1,9 +1,12 @@
 // here we define all settings that have an effect on how the pareto chart will be rendered and look like,
 
+import { Tooltip } from "spotfire-api";
```

Comment:
I wonder if this is possible to do in index.ts instead?

Reply:
> I wonder if this is possible to do in index.ts instead?

I think as setting.js has been imported in the index.js we can have it in setting as well! But maybe to be more consistent with index.js we can move it

Reply:
I tried and I think the only way is if you then make the type any. But I am not sure that is better. Since this is only importing the tooltip interface i think it is okay to leave it like this.

Reply:
@AstridBer, I think you're right.